ci-operator/step-registry/openstack/conf/generateconfig/openstack-conf-generateconfig-commands.sh

```diff
@@ -4,8 +4,10 @@ set -o nounset
 set -o errexit
 set -o pipefail
 
-#Read necessary variables
-CLUSTER_NAME=$(<"${SHARED_DIR}"/CLUSTER_NAME)
+CLUSTER_NAME=$(<"${SHARED_DIR}/CLUSTER_NAME")
+OPENSTACK_EXTERNAL_NETWORK=$(<"${SHARED_DIR}/OPENSTACK_EXTERNAL_NETWORK")
+OPENSTACK_COMPUTE_FLAVOR=$(<"${SHARED_DIR}/OPENSTACK_COMPUTE_FLAVOR")
```

Comment:
```suggestion
PROFILE_OPENSTACK_COMPUTE_FLAVOR=$(<"${SHARED_DIR}/OPENSTACK_COMPUTE_FLAVOR")
OPENSTACK_COMPUTE_FLAVOR=${OPENSTACK_COMPUTE_FLAVOR:-PROFILE_OPENSTACK_COMPUTE_FLAVOR}
```
to allow overriding a profile via env vars

Reply:
This already works!

The openstack-conf-resourcenames accepts both OPENSTACK_COMPUTE_FLAVOR and OPENSTACK_EXTERNAL_NETWORK and will obey them as overrides:

https://github.com/openshift/release/pull/19081/files#diff-42ad3a59285422a95b3e6c61abf1380bb0e59c4b943ca2959ee9dcb07ac2f175R17

Reply:
Oh, wait, I know what you mean.

You specified "via env vars"; meaning that the steps should obey step-specific env vars, so that each step can be reused in a workflow that doesn't use `openstack-conf-resourcenames`.

I now understand your point and I actually agree. However, from our perspective, the need for switching clouds happens weekly. The need for switching flavors happened so far exactly zero times. So while I agree, I would tend to fix that in a follow-up, with something very similar to the solution you suggest. Deal?